install/install.py

```diff
@@ -62,7 +62,8 @@ def check_sudo():
         if sudo == 0:
             return
         else:
-            print("[WARNING] Your user does not have sudo priveleges. Some OWTF components require sudo permissions to install")
+            Colorizer.warning("[!] Your user does not have sudo privileges. "
```

Comment:
Sure, I will do it right away. My Pycharm editor automatically performs Soft Wraps. I will redo these two lines to a single line.